### PR TITLE
Feature/re arrange report landing page

### LINKED ISSUE #58: Rearrange report page

### PR DESCRIPTION
**Pull request by jofomah**
fixes #58

### CODE REVIEW

File: src/app/reports/invoice/invoice.html [hunk 1]
```diff
@@ -16,13 +16,13 @@ <h3 class="panel-title">Delivery Rounds</h3>
       </tr>
     </thead>
     <tbody>
-      <tr ng-repeat="round in reportsCtrl.deliveryRounds">
+      <tr ng-repeat="round in reportsAllCtrl.deliveryRounds">
```
Comment (freefony) [inline src/app/reports/invoice/invoice.html:19]: if the `reportsAllCtrl.deliveryRounds` is a large array we might want to add `track by` to `ng-repeat` for speed

Comment (jofomah) [inline src/app/reports/invoice/invoice.html:19]: @freefony noted, like i said, i just moved this file, and i think we already have a ticket for paginating and optimising this invoice table Haykel created.

Comment (freefony) [inline src/app/reports/invoice/invoice.html:19]: ok i ll go ahead and merge this, so the pagination can take care of it

Comment (jofomah) [inline src/app/reports/invoice/invoice.html:19]: actually this page will likely change before next deployment, have already discussed with Janet about pagination and other optimization (including track by etc).

**Pull request merged**